### DISCUSSION TITLE
Timetable on AWS WAF/Shield?

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#569] [Help] [answered]
Are there any roadmap updates on a Terraform module for AWS WAF?

---

<ins datetime="2022-09-29T18:34:22Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109302">Tracked in ticket #109302</a></p>
</ins>

**Comment by rhoboat** [comment]
Point of clarification, which WAF are you referring to? (Expand the acronym.)

**Comment by rhoboat** [comment, marked as answer]
Assuming Web Application Firewall, unfortunately we do not have a module for managing AWS WAF in our library at the moment. We don't have it on our roadmap at the moment, but we will add it to our planning discussions.

**Reply by iantybo** [reply]
Yup, that's it!
